firebaseui/resources/deps.cljs

```diff
@@ -0,0 +1,3 @@
+{:foreign-libs [{:file "cljsjs/development/firebaseui.inc.js"
```

Comment:
Instead of  creating `deps.cljs` by hand, it is preferred to use `deps-cljs` task from boot-cljsjs. It will make  set up the paths automatically (and ensure they are correct).

`(deps-cljs :name "cljsjs.firebaseui")`